mappings/net/minecraft/client/input/KeyboardInput.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/unmapped/C_cxdvjlve net/minecraft/client/input/KeyboardInput
-	FIELD f_cgjjsojv settings Lnet/minecraft/unmapped/C_lscptbhn;
+	FIELD f_cgjjsojv options Lnet/minecraft/unmapped/C_lscptbhn;
+	FIELD f_isclmpwk SLOW_DOWN_FACTOR D
```

Comment:
This field was removed, and it causes an NPE in the build action

Reply:
Oh, I'll fix that